docs/api/getting-started.md

```diff
@@ -0,0 +1,47 @@
+---
+slug: /
+sidebar_position: 1
+---
+
+# Getting Started with the API
+
+Welcome to the Saiki API. This guide will walk you through the essential first steps to begin interacting with your Saiki agent programmatically.
+
+## 1. Starting the API Server
+
+Before you can make any API calls, you must start the Saiki server. This single command enables both the REST and WebSocket APIs.
+
+Run the following command in your terminal:
+
+```bash
+saiki --mode server
+```
+
+By default, the server will run on port `3001`. You should see a confirmation message in your terminal indicating that the server has started successfully.
+
```

Comment:
we can add a small section here about using their own config file in server mode so they get an idea of how it relates to their agent

a mini tutorial of using the API could also be helpful. maybe a better fit for the tutorials section but yeah then we can link people there. not super high priority